src/avatar.h

```diff
@@ -30,6 +30,13 @@ class mission;
 class monster;
 class npc;
 
+enum avatar_strikemode : int {
```

Comment:
You should convert this to an enum class and remove the prefixes from the members.
```suggestion
enum class avatar_strikemode : int {
```

Reply:
I have changed the enum to an enum class, removed the prefixes, and applied the relevant changes where applicable. Tested to make sure it still works just fine :)